app/javascript/components/navbar.js

```diff
@@ -0,0 +1,14 @@
+const initUpdateNavbarOnScroll = () => {
+  const navbar = document.querySelector('.navbar-lewagon');
+  if (navbar) {
+    window.addEventListener('scroll', () => {
+      if (window.scrollY >= window.innerHeight) {
```

Comment:
@zuccamia Okay so the reason why the JS doesn't seem to be running is because the page isn't long enough for it to appear, but it actually works. Open inspect on a browser and you will be able to see it. Navbar color changes from white to dark blue at the moment so not being able to see the mommy photo doesn't change. 

What we need to think about more is that even if a user scrolls down, they will see the navbar always at the top, so they can use it without scrolling up again. (it's not JS, it's bootstrap fixed-top)